build.gradle

```diff
@@ -0,0 +1,354 @@
+    repositories {
+        mavenDeployer {
+            beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
+
+            repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
+                authentication(userName: project.findProperty("sonatypeUsername"), password: project.findProperty("sonatypePassword"))
+            }
+
+            snapshotRepository(url: "https://artifactory.broadinstitute.org/artifactory/libs-snapshot-local/") {
+                authentication(userName: System.env.ARTIFACTORY_USERNAME, password: System.env.ARTIFACTORY_PASSWORD)
+            }
+
+            pom.project {
+                name 'Picard'
+                packaging 'jar'
+                description 'A set of command line tools (in Java) for manipulating high-throughput sequencing (HTS) data and formats such as SAM/BAM/CRAM and VCF.'
+                url 'http://broadinstitute.github.io/picard/'
+
+                developers {
+                    developer {
+                        id 'picard'
+                        name 'Picard Team'
+                        url 'http://broadinstitute.github.io/picard'
+                    }
+                }
+
+                scm {
+                    url 'git@github.com:broadinstitute/picard.git'
+                    connection 'scm:git:git@github.com:broadinstitute/picard.git'
+                }
+
+                licenses {
+                    license {
+                        name 'MIT License'
+                        url 'http://opensource.org/licenses/MIT'
+                        distribution 'repo'
+                    }
+                }
+            }
+        }
+    }
+    doFirst {
+        System.out.println("Uploading version $version")
+    }
+}
+
+task copyJavadoc(dependsOn: 'javadoc', type: Copy) {
+    from 'build/docs/javadoc'
+    into 'javadoc'
+}
+
+task updateGithubDocumentation(dependsOn: 'copyJavadoc') << {
+    def grgit = Grgit.open(project.projectDir)
+    grgit.branch.remove(names:['gh-pages'], force: true)
+    grgit.checkout(branch: 'gh-pages', startPoint: 'origin/gh-pages', createBranch: true)
+    //add javadoc files
+    grgit.add(patterns: ['javadoc'])
+
+    println("Updating javadoc for release: $version")
+
+    grgit.commit(message: "Updating javadoc for release: $version", all: true)
+
+    grgit.push()
+    grgit.checkout(branch: 'master')
+}
+
+task copyStaticWebFiles(dependsOn: 'documentAll', type: Copy) {
+    from 'dist/html'
+    into '_includes'
+}
+
+task updateWebsite(dependsOn: 'copyStaticWebFiles') << {
+    def grgit = Grgit.open(project.projectDir)
+    grgit.branch.remove(names:['gh-pages'], force: true)
+    grgit.checkout(branch: 'gh-pages', startPoint: 'origin/gh-pages', createBranch: true)
+    //add javadoc files
+    grgit.add(patterns: ['javadoc'])
+
+    println("Adding website files for release: $version")
+
+    grgit.commit(message: "Adding website files for release: $version", all: true)
```

Comment:
Is this going to be a complete replacement for the release script, or will that still be needed?